crates/gpui/src/platform/windows/window.rs

```diff
@@ -40,15 +40,12 @@ use crate::*;
 
 pub(crate) struct WindowsWindowInner {
     hwnd: HWND,
-    origin: Cell<Point<GlobalPixels>>,
-    size: Cell<Size<GlobalPixels>>,
-    mouse_position: Cell<Point<Pixels>>,
```

Comment:
I believe this property existed so we didn't have to use win32 api calls everytime we wanted to know where the mouse position is. I don't know how much of a big deal that is, but that's probably something we want to preserve.

I'm pretty sure `GetCursorPos` is thread safe, but im sure having our own local value is faster.

Thoughts anyone?

Reply:
I noticed that this variable is only used when calling `window::mouse_position`. Upon tracing this function, I found that it is only called once when `Zed` is opened. I am not sure of the purpose of this function, but I suspect it may soon be deprecated. Given the low frequency of calls, I decided to remove this variable.

Reply:
oh you're right! Looks like `Window` tracks the mouse position itself already.

In that case this seems fine to me!